utils/ansible/ftest/templates/daos-make.sh.j2

```diff
@@ -196,6 +208,18 @@ if [[ ! -d "$DAOS_INSTALL_DIR" ]] ; then
 	fi
 fi
 
+if [[ ! -f "$DAOS_INSTALL_DIR/bin/activate" ]] ; then
+	info "Installing avocado"
+	run $PYTHON_EXE -m venv "$DAOS_INSTALL_DIR"
+	source "$DAOS_INSTALL_DIR/bin/activate"
+	run $PIP_EXE install --upgrade pip
+	run $PIP_EXE install -r "$DAOS_SOURCE_DIR/requirements.txt"
+	deactivate
+	info "Customize venv for DAOS install"
+	run rm -fv "$DAOS_INSTALL_DIR/lib64"
+	run mkdir -pv "$DAOS_INSTALL_DIR/lib64"
+fi
+
 cd "$DAOS_SOURCE_DIR"
 SCONS_OPTS="-j $JOBS_NB --implicit-deps-changed"
```

Comment:
FYI - scons has a `-C <directory>` option you could use here to avoid the use of `cd`

Reply:
good to know, thanks for the tips.

- [ ] Use scons with -C option

Reply:
> good to know, thanks for the tips.
> 
> * [x]  Use scons with -C option

Fixed with commit bce8c6cb288ad0605a53f42649b035f6e81d8546